scripts/languageserver/provider_misc.jl

```diff
@@ -10,6 +10,7 @@ const serverCapabilities = ServerCapabilities(
                         SignatureHelpOptions(["("])) 
 
 function process(r::JSONRPC.Request{Val{Symbol("initialize")},Dict{String,Any}}, server)
+    server.rootPath=haskey(r.params,"rootPath") ? "file://"*r.params["rootPath"] : ""
```

Comment:
Maybe not use the ``file://`` prefix here, after all this is not a file URI? Maybe just store the path as we get it?